Content.Client/AirlockPainter/AirlockPainterSystem.cs

```diff
@@ -0,0 +1,54 @@
+using Content.Shared.AirlockPainter;
+using Robust.Client.Graphics;
+using Robust.Client.ResourceManagement;
+using Robust.Shared.Utility;
+using System.Linq;
+using static Robust.Shared.GameObjects.SharedSpriteComponent;
+
+namespace Content.Client.AirlockPainter
+{
```

Comment:
FIY you can also use file scoped namespaces. AFAIK not really a convention listed anywhere, but IMO its nicer & leads to files with less indentation.